### DISCUSSION TITLE
Node Red - Button issue

### DISCUSSION THREAD

**Discussion by Moach2000** [zachowj/node-red-contrib-home-assistant-websocket#1277] [Q&A]
Dear Community!
I updated HASSIO yesterday and I'm not able to use Buttons anymore.
Core 2024.1.5
Supervisor 2023.12.1
OS 11.4

Node Red 17.0.3
Node Red contrib home assistant WebSocket 0.62.3

I created within Node Red a button (as always), used that within the Lovelace but I always get - a kind of message - no service, no possible to call the service for button/....

Any ideas?
Br

**Comment by zachowj** [comment]
What version of the [Node-RED companion component](https://github.com/zachowj/hass-node-red) are you running? The version info can be found from the [diagnostic button](https://zachowj.github.io/node-red-contrib-home-assistant-websocket/guide/diagnostics.html).

**Reply by Moach2000** [reply]
Hi! I tried to get out the data (as mentioned in the documentation - both ways) without any success.
Via Klick&Fetch with the buttons - system always stucks within the status "Fetching data..."
I also tried to use the other way (copied the example) and did a test without any positive result.....
Br, Christian

**Reply by Moach2000** [reply]
One more fb - I also did a text an on other HASSIO instance with success (but an older setup). Still the case that buttons (on lovelace) dont work as before.

**Reply by Moach2000** [reply]
And one more - I got an 401 auth. error (for wathever reasons...)

**Reply by Moach2000** [reply]
Works now. I didn't change anything - seems to work again. Thread can be closed.
